docs/v1/manual.md

```diff
@@ -221,7 +221,7 @@ func main() {
   app := cli.NewApp()
 
   app.Flags = []cli.Flag {
-    cli.StringFlag{
+    &cli.StringFlag{
```

Comment:
This is a difference between the `v1` and `v2` versions of the API. These are the docs for `v1`, in which the non-pointer version of the `cli` flags implement the interface.

Reply:
I met this bug with cli v1.22.4 with go 1.14.
I can show you all of source code, go.mod everything I used.

Where should I submit a full bug template to this PR or issues?

Thanks for your feedback.

Reply:
GitHub links are weird, but I think this is it: https://github.com/urfave/cli/issues/new?assignees=&labels=kind%2Fbug%2C+status%2Ftriage%2C+area%2Fv1&template=v1-bug-report.md&title=v1+bug%3A+%28+your+bug+title+goes+here+%29

If that doesn't work, on this repo, click "New Issue" > "v1 bug report", and the template should be filled out for you

Reply:
I missed it. You're right. Without Address-of-Operator it does work. 
go run main.go / go build main.go both is OK, but my IDE GoLand still show me syntax error.

Anyway Thanks for your feedback. 
It does work. Thanks.